Algorithm.CSharp/QuantConnect.Algorithm.CSharp.csproj

```diff
@@ -12,6 +12,8 @@
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <LangVersion>5</LangVersion>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
```

Comment:
I dont think so. I think it was added automatically upon adding the NuGet package